deploy/helm/scf/assets/operations/instance_groups/diego-cell.yaml

```diff
@@ -1,6 +1,6 @@
 # Selectively remove jobs temporarily.
-- type: remove
-  path: /instance_groups/name=diego-cell/jobs/name=rep
+# - type: remove
```

Comment:
We should remove this completely as we are enabling rep in this PR definitively.